rare/components/tabs/games/__init__.py

```diff
@@ -313,32 +330,32 @@ def get_visibility(widget):
             self.installing_widget.setVisible(get_visibility(self.installing_widget))
 
     def update_list(self, app_names: list = None):
-        logger.debug("Updating list for " + str(app_names))
+        logger.debug(f"Updating list for {str(app_names)}")
```

Comment:
Nit: I might actually stop joking now and mark all of these just as "Nit: no need for that `str`"